components/sup/src/manager/mod.rs

```diff
@@ -291,18 +309,15 @@ impl Manager {
         Ok(())
     }
 
-    pub fn remove_service(&self, service: &Service) -> Result<()> {
-        // TODO FN: there is more to removing a service, more to follow
-        service.remove()?;
+    pub fn remove_service(&self, service: &mut Service) -> Result<()> {
```

Comment:
Ya we can do that, let's hit it in another refactor if we feel the words should be the same. It's cool either way with me